test/test_client.py

```diff
@@ -313,6 +328,8 @@ def raise_for_status(self):
 
 class TestMakeHttpRequest(ClientTest):
 
+    apiPath = 'https://tc-tests.localhost/api/fake/v1/test'
```

Comment:
Since this is the same base url, maybe we can use the rootUrl from the TCTest value?  Either way, like in that file, we should use a sub-domain of `example.com` or some similar.